packages/config/config/devices/0x0330/zv2835rac-nf.json

```
@@ -0,0 +1,271 @@
+	"description": "Knob Smart Dimmer",
+	"devices": [
+		{
+			"productType": "0x0200",
+			"productId": "0xd017"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"2": {
+			"label": "State After Power Failure",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 2,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 1
+				},
+				{
+					"label": "Previous State",
+					"value": 2
+				}
+			]
+		},
+		"3": {
+			"label": "Send Basic Report",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		"4": {
+			"label": "Default fade time",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Instantly",
+					"value": 0
+				},
+				{
+					"label": "from 1 to 127 seconds",
+					"value": 1
+				},
+				{
+					"label": "1 to 127 minutes",
+					"value": 80
+				},
+				{
+					"label": "Factory default duration. (3 second)",
+					"value": 255
+				}
+			]
+		},
+		"5": {
+			"label": "Minimum brightness value",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 50,
+			"defaultValue": 0
+		},
+		"6": {
+			"label": "Maximum brightness value",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 100
+		},
+		"7": {
+			"label": "MOSFET driving type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Trailing edge",
+					"value": 0
+				},
+				{
+					"label": "Leading edge",
+					"value": 1
+				}
+			]
+		},
+		"8": {
+			"label": "Step of knob",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 50,
+			"defaultValue": 5
+		},
+		"9": {
+			"label": "Added to and removed from a network",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		"10": {
+			"label": "Detect load",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				},
+				{
+					"label": "Only after first re-power",
+					"value": 2
+				}
+			]
+		},
+		"11": {
+			"label": "Wiring type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"readOnly": true
+		},
+		"12": {
+			"label": "Load type",
```

```suggestion
			"label": "Load Type",
```
Any explanation what these types mean?